src/main/java/edu/greenblitz/tobyDetermined/OI.java

```diff
@@ -1,10 +1,19 @@
 package edu.greenblitz.tobyDetermined;
```

Comment:
Dont merge with those changes

Reply:
what?

Reply:
don't merge with the changes to the buttons, delete your button changes or if you want - put them on a 3rd\4th controller (other then the main and second controllers).